### ISSUE TITLE
Wrong de localization lat/lang in ContactBundle

### ISSUE THREAD

**Issue by Skullfox** [sulu/sulu#6896]
| Q | A
| --- | ---
| Sulu Version | 2.2+

# Actual Behavior

Latitude/Longitude are switched in DE

```
    "sulu_contact.latitude": "Längengrad",
    "sulu_contact.longitude": "Breitengrad",
```

should be 

    "sulu_contact.latitude": "Breitengrad",
    "sulu_contact.longitude": "Längengrad",

# Possible Solutions
[/src/Sulu/Bundle/ContactBundle/Resources/translations/admin.de.json#L74](https://github.com/sulu/sulu/blob/2.5/src/Sulu/Bundle/ContactBundle/Resources/translations/admin.de.json#L74)

```
    "sulu_contact.latitude": "Breitengrad",
    "sulu_contact.longitude": "Längengrad",
```

**Comment by alexander-schranz** [comment]
@Skullfox Thx for creating the issue. Can you open a Pull Request for it. You just need to target the current `2.4` Branch here: https://github.com/sulu/sulu/blob/2.4/src/Sulu/Bundle/ContactBundle/Resources/translations/admin.de.json

**Comment by alexander-schranz** [comment]
Fixed in #6963
